connect/transforms/src/test/java/org/apache/kafka/connect/transforms/MaskFieldTest.java

```diff
@@ -153,4 +179,56 @@ public void withSchema() {
         assertEquals(Collections.emptyMap(), updatedValue.get("map"));
     }
 
+    @Test
+    public void schemalessWithReplacement() {
+        checkReplacementSchemaless(singletonList("short"), (short) 123);
+        checkReplacementSchemaless(singletonList("byte"), (byte) 123);
+        checkReplacementSchemaless(singletonList("int"), 123);
+        checkReplacementSchemaless(singletonList("long"), 123L);
+        checkReplacementSchemaless(singletonList("float"), 123.0f);
+        checkReplacementSchemaless(singletonList("double"), 123.0);
+        checkReplacementSchemaless(singletonList("string"), "123");
+        checkReplacementSchemaless(singletonList("bigint"), BigInteger.valueOf(123L));
+        checkReplacementSchemaless(singletonList("bigdec"), BigDecimal.valueOf(123.0));
```

Comment:
There are lots of occurrences of `checkReplacementSchemaless(singletonList(`. Perhaps it would make sense to create a second `checkReplacementSchemaless(String replacement, Object value)` that returned `checkReplacementSchemaless(singletonList(replacement), value)`, and then most of these lines could be simplified a bit.

Same for `checkReplacementWithSchema(...)`.